Rakefile

```diff
@@ -14,6 +14,13 @@
 
 require 'bundler/gem_tasks'
 require 'rspec/core/rake_task'
+require 'zipkin-tracer'
+require 'benchmark'
+require 'benchmark/ips'
+require 'faraday'
+require 'rack/mock'
+require 'rbtrace'
+require 'tempfile'
```

Comment:
what about using a group in the Gemfile so you could just do `Bundle.require(:performance)` for instance?